Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -699,17 +700,21 @@ private class ModuleAliasTracker {
     // This func should be called once per product
     func trackTargetsPerProduct(product: Product,
                                 package: PackageIdentity) {
-        let productTargetDeps = product.targets.map{$0.dependencies}.flatMap{$0}
-        for dep in productTargetDeps {
+        let targetDeps = product.targets.map{$0.dependencies}.flatMap{$0}
+        var allTargetDeps = product.targets.map{$0.dependentTargets().map{$0.dependencies}}.flatMap{$0}.flatMap{$0}
```

Comment:
I think you can use `recursiveTargetDependencies` here?

Reply:
product and target here are non-resolved ones, so that method is not available.